docs/DeveloperGuide.md

```diff
@@ -223,14 +250,14 @@ shown in the activity diagram below.
     -   Cons: There is a need to clear the list before adding the selected company to the `UniqueCompanyList` to ensure that only one company is displayed in the `CompanyDetailPanel` at any time.
 
 -   **Alternative 2:** Create a new `Company` object in `AddressBook` to store the selected company which the user wishes to view.
-    -   Pros: It is intuitive to create a new `Company` object to store the selected company which the user wishes to view.
+    -   Pros: There is not a need to clear the list as there is not list involved. 
```

Comment:
Suggestion: Since there are no lists involved, there isn't a need to clear the list.